### PR TITLE
Add Dockerfile and workflows to build images

### LINKED ISSUE #11: Dockerイメージ提供

### PR DESCRIPTION
**Pull request by tsuzu**
## WHY
I'd like to use this on Kubernetes!

## WHAT
Closes #11

- Add Dockerfile
- Add a workflow to build images

## QA
https://github.com/tsuzu/annict-epgstation-connector/actions/runs/5414716816/jobs/9842163134

```
tsuzu@Tsuzus-MacBook-Pro annict-epgstation-connector % docker run -ti --rm ghcr.io/tsuzu/annict-epgstation-connector:sha-450bf23 --help
Unable to find image 'ghcr.io/tsuzu/annict-epgstation-connector:sha-450bf23' locally
sha-450bf23: Pulling from tsuzu/annict-epgstation-connector
0b41f743fd4d: Already exists 
fe5ca62666f0: Already exists 
b02a7525f878: Already exists 
fcb6f6d2c998: Already exists 
e8c73c638ae9: Already exists 
1e3d9b7d1452: Already exists 
4aa0ea1413d3: Already exists 
7c881f9ab25e: Already exists 
5627a970d25e: Already exists 
34fb4f507127: Pull complete 
Digest: sha256:572262d8437f7f17ceeb271b9f4e3829a94f565bda162277e3a1094da07abaa7
Status: Downloaded newer image for ghcr.io/tsuzu/annict-epgstation-connector:sha-450bf23
NAME:
   annict-epgstation-connector - generate recording rules based on the annict statuses

USAGE:
   annict-epgstation-connector [global options] command [command options] [arguments...]

COMMANDS:
   sync     sync
   help, h  Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --annict-api-token value     An API token of annict GraphQL API server [$ANNICT_API_TOKEN]
   --epgstation-endpoint value  An endpoint of EPGStation API server [$EPGSTATION_ENDPOINT]
   --help, -h                   show help
tsuzu@Tsuzus-MacBook-Pro annict-epgstation-connector % 
```

### CODE REVIEW

**Comment by tsuzu** [comment]
Github Actions don't run....

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      - "v[0-9]+.[0-9]+.[0-9]+"
+    branches: 
+      - master
+  pull_request:
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
```
Comment (tsuzu) [inline .github/workflows/docker.yml:15]: to push images to ghcr.io

**Comment by tsuzu** [comment]
Should disable pushing images on PRs?

**Comment by musaprg** [comment]
I'd say this should be disabled for now. I'm going to check later if it's possible or not 🙂

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+name: Build container images
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+    branches: 
+      - master
+  pull_request:
```
Comment (musaprg) [inline .github/workflows/docker.yml:9]:
```suggestion
```

**Review by musaprg — CHANGES_REQUESTED** [review]
Pushing GHCR for PR isn't working properly due to a token issue. Could you remove it for now?

**Review by musaprg — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**